demisto_sdk/commands/test_content/test_modeling_rule/test_modeling_rule.py

```diff
@@ -1279,16 +1280,26 @@ def test_modeling_rule(
     handle_deprecated_args(ctx.args)
 
     logger.info(
-        f"[cyan]Modeling Rules directories to test: {inputs}[/cyan]",
+        "[cyan]Test Modeling Rules directories to test:[/cyan]",
```

Comment:
??? What are the directories to test?
Did you delete `{inputs}` by mistake?

Reply:
i'm printing them one per line, in the next lines. make it more readable in the build logs